### PR TITLE
Bump alpine to v3.14.0

### PR DESCRIPTION
**Pull request by wozniakjan**
**Description**

Changes proposed in this pull request:

- Bumping alpine and rebuilding the image updates curl to `7.77.0` to address a CVEs present in `7.76.1`.

**Related issue(s)**

* https://github.tools.sap/kyma/security-scans/issues/8630
* https://github.tools.sap/kyma/security-scans/issues/8627
* https://github.tools.sap/kyma/security-scans/issues/8628

### CODE REVIEW

**Comment by wozniakjan** [comment]
https://protecode.c.eu-de-2.cloud.sap/#/product/660746/analysis
https://protecode.c.eu-de-2.cloud.sap/#/product/656111/analysis 
fwiw both seem to be containing curl `v7.77.0-r1` which seem to address the CVEs

File: resources/kcp/values.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ global:
       version: "PR-723"
```
Comment (ksputo) [inline resources/kcp/values.yaml:18]: if you're bumping this base image: https://github.com/kyma-project/control-plane/blob/b267affed5890860795538b9be57ee6c7b4cd6e4/components/kyma-environment-broker/Dockerfile.cleanup

You'd like to bump the cleanup job too.

```
Dockerfile.sac -> kyma_environments_subaccount_cleanup_job
Dockerfile.cleanup -> kyma_environments_cleanup_job
```

```suggestion
      version: "PR-857"
```

Comment (wozniakjan) [reply resources/kcp/values.yaml:18]: thanks!

**Review by ksputo — COMMENTED** [review]
/lgtm

**Review by ksputo — APPROVED** [review]

**Pull request merged**